### PR TITLE
inotify-info: unstable-2024-01-05 -> 0.0.1

### PR DESCRIPTION
**Pull request by motiejus**
## Description of changes

Bump version. No changes in functionality, just started creating releases.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)

### CODE REVIEW

File: pkgs/by-name/in/inotify-info/package.nix [hunk 1]
```diff
@@ -1,28 +1,31 @@
...
+  stdenv.mkDerivation (finalAttrs: {
+    name = "inotify-info";
+    inherit (version);
 
-stdenv.mkDerivation (finalAttrs: {
-  pname = "inotify-info";
-  version = "unstable-2024-01-05";
+    src = fetchurl {
+      url = "https://github.com/mikesart/inotify-info/releases/download/v${version}/inotify-info-${version}.tar.gz";
+      sha256 = "sha256-SDE0Y+HNyQPbl+Mh6HBe2LXeEd8TyfStrJgzSxxLJDM=";
```
Comment (kirillrdy) [inline pkgs/by-name/in/inotify-info/package.nix:14]:
```suggestion
      hash = "sha256-SDE0Y+HNyQPbl+Mh6HBe2LXeEd8TyfStrJgzSxxLJDM=";
```

File: pkgs/by-name/in/inotify-info/package.nix [hunk 2]
```diff
@@ -1,28 +1,31 @@
...
+  version = "0.0.1";
+in
+  stdenv.mkDerivation (finalAttrs: {
+    name = "inotify-info";
+    inherit (version);
 
-stdenv.mkDerivation (finalAttrs: {
-  pname = "inotify-info";
-  version = "unstable-2024-01-05";
+    src = fetchurl {
```
Comment (kirillrdy) [inline pkgs/by-name/in/inotify-info/package.nix:12]: can we go back to fetchFromGithub ?

Comment (motiejus) [reply pkgs/by-name/in/inotify-info/package.nix:12]: Github [does not promise to keep the generated hashes stable](https://github.blog/2023-02-21-update-on-the-future-stability-of-source-code-archives-and-hashes/). That is the reason why I generated the release archives in the upstream project.

`fetchFromGitHub` relies on those hashes unchanged.

Comment (kirillrdy) [reply pkgs/by-name/in/inotify-info/package.nix:12]: @motiejus the article you have linked is not related to git commit revisions

File: pkgs/by-name/in/inotify-info/package.nix [hunk 3]
```diff
@@ -1,14 +1,17 @@
...
-  version = "unstable-2024-01-05";
+  name = "inotify-info";
+  version = "0.0.1";
 
   src = fetchFromGitHub {
     owner = "mikesart";
     repo = "inotify-info";
-    rev = "a7ff6fa62ed96ec5d2195ef00756cd8ffbf23ae1";
-    hash = "sha256-yY+hjdb5J6dpFkIMMUWvZlwoGT/jqOuQIcFp3Dv+qB8=";
+    rev = "6abef0e9e7638d16bbe59cfd88c72b680699fe34";
```
Comment (NickCao) [inline pkgs/by-name/in/inotify-info/package.nix:13]:
```suggestion
    rev = "refs/tags/v${finalAttrs.version}";
```

**Review by nicoonoclaste — CHANGES_REQUESTED** [review]
Looks good, I only have pretty-minor feedback

**Review by nicoonoclaste — APPROVED** [review]

File: pkgs/by-name/in/inotify-info/package.nix [hunk 4]
```diff
@@ -22,7 +25,7 @@ stdenv.mkDerivation (finalAttrs: {
     homepage = "https://github.com/mikesart/inotify-info";
     license = licenses.mit;
     mainProgram = "inotify-info";
-    maintainers = with maintainers; [ motiejus ];
+    maintainers = with maintainers; [motiejus];
```
[the remark this answers is not in the document]
Comment (motiejus) [reply pkgs/by-name/in/inotify-info/package.nix]: This was automatically formatted with alejandra formatter.

**Pull request merged**